src/app/static/src/app/hintVersion.ts

```diff
@@ -1 +1 @@
-export const currentHintVersion = "3.1.0";
```

Comment:
Arguably all of these changes don't merit a version change since they aren't changing the publicly available behaviour of the app. And the third number is generally for patches (bug fixes) anyway, I think? We can put in a minor change when we flip the switch. Might cut down on merge conflicts a bit..

Reply:
ohhh that actually helps a lot, given how many hint PRs there are, i was wondering how to efficiently manage version changes, will remove the change!